internal/router/router_test.go

```diff
@@ -90,3 +106,13 @@ func Test_JenkinsAPI_routes_are_setup(t *testing.T) {
 	mockedRouter.ServeHTTP(w, req)
 	require.Equal(t, "{\"data\":{\"state\":\"\"}}\n", w.GetBody(), "Routing failed for /api/jenkins/start")
 }
+
+func Test_JenkinsAPI_status_is_fetched(t *testing.T) {
+	mockedJenkinsAPI := &mockJenkinsAPI{}
```

Comment:
is this method testing anything more than the **mock** jenkins api?

Reply:
Yes, I think it's only testing `MockJenkinsAPI` . But I saw a similar implementation for `/api/jenkins/start` so I wrote it. I can remove it if you don't think it's required.